Automattic/pocket-casts-ios
# Icon: Halloween

Happy Halloween!

This PR adds a new icon for the Halloween season.

| iPhone | iPad |
| ------ | ----- |
| ![Simulator Screen Shot - iPhone 14 - 2022-10-11 at 10 14 57](https://user-images.githubusercontent.com/7040243/195100903-54e7e389-4c42-438c-a9c2-ad454c7c618e.png) | ![Simulator Screen Shot - iPad Pro (12 9-inch) (5th generation) - 2022-10-11 at 10 14 55](https://user-images.githubusercontent.com/7040243/195100955-6cdf41c5-e9c7-40f4-9976-6ab6971ef8dd.png) |

## To test

1. Run the app and login with a plus account
2. Go to Profile > Settings > Appearance and select the "Halloween" icon
3. ✅ Make sure the icon changes

Please test on iPhone and iPad.

## Checklist
